### PR TITLE
New Drug - Ritalin

### PR DESCRIPTION
**Pull request by RichardBlonski**
#### Summary
```SUMMARY: Content "Adds a new drug named Ritalin"```

#### Purpose of change
``` Adding a new drug named "Ritalin" , it's a drug used to treat attention deficit hyperactivity disorder  (ADHD), in game sence it makes you more forcused (buff int & per) however like in real-life you will feel more weaker debuffing (strg & dex).```

#### Describe alternatives you've considered

I've considered attempting to make the drug more real-life however I couldn't figure out how to code into the file a way to make the player avoid hunger -
(like still getting hungry but won't be notified about it until the drug effects end).  

#### Additional context

```Made it look like antibiotics so it would work with tilesets```

### CODE REVIEW

**Comment by LovamKicsiGazsii** [comment]
Use this to lint your files --> http://dev.narc.ro/cataclysm/format.html

File: data/json/effects.json [hunk 1]
```diff
@@ -1659,5 +1659,18 @@
   {
     "type": "effect_type",
     "id": "has_prospectus"
+  },
+  {
+	"type": "effect_type",
+	"id": "ritalin",
+	"name": ["Used Methylphenidate", "Used Ritalin"],
+	"desc": ["You feel more focused and weak"],
```
Comment (Night-Pryanik) [inline data/json/effects.json]:
```suggestion
	"desc": ["You feel more focused and weak."],
```

File: data/json/effects.json [hunk 2]
```diff
@@ -1659,5 +1659,18 @@
...
+	"max_intensity": 1000,
+	"int_dur_factor": 1,
+	"base_mods": {"dex_mod": [-3], "int_mod": [4], "per_mod": [3], "str_mod": [-4] },
+	"apply_message": "You feel weak and focused",
```
Comment (Night-Pryanik) [inline data/json/effects.json]:
```suggestion
	"apply_message": "You feel weak and focused.",
```

File: data/json/effects.json [hunk 3]
```diff
@@ -1659,5 +1659,18 @@
...
+	"miss_messages": [ [ "You feel weak", 1] ]
```
Comment (Night-Pryanik) [inline data/json/effects.json]:
```suggestion
	"miss_messages": [ [ "You feel weak.", 1] ]
```

**Comment by golankra123** [comment]
> Use this to lint your files --> http://dev.narc.ro/cataclysm/format.html

I've used it - however, in the med.json most code is labeled around the same as I did it

**Comment by cafeoh** [comment]
I can't say I've experienced something akin to reduced strength or agility for the brief time I've used Ritalin myself, and haven't really seen that listed as a side-effect online. Disorientation might occur on too high doses (if that's what you're referring to), but a standard dosage shouldn't be too dissimilar from a strong coffee buzz for a long period of time, and generally very similar to other amphetamines-like stimulants (like adderall or meth).

Regarding hunger it might negate some of the side effects of hunger, but (like most other uppers) it really only distracts you from it, meaning that you can still tell if you're hungry or not (especially if you're actually famished) but it might displace the morale malus.

This might be up for debate but in general I don't think there is a very meaningful difference between Ritalin and Adderall, and like a lot of pharmaceutical drugs and street counterparts, the main difference seems to be history, reputation and ease of abuse (plus the anecdotal conflict or synergy in some individuals)

**Comment by golankra123** [comment]
**Close Due :** Feedback 🥇

**Pull request closed without merging**